### PR TITLE
Update f5_rest_cli.py

### PR DESCRIPTION
**Pull request by gabetellier**
Mege timeout parameter description changes requested and reviewed by Patrick Gouband.

### CODE REVIEW

File: ansible_collections/f5networks/f5_modules/plugins/doc_fragments/f5_rest_cli.py [hunk 1]
```diff
@@ -51,9 +51,18 @@ class ModuleDocFragment(object):
...
         description:
+          - transport: rest
           - Specifies the timeout in seconds for communicating with the network device
-            for either connecting or sending commands.  If the timeout is
+            for either connecting or sending commands. If the timeout is
             exceeded before the operation is completed, the module will error.
+          -
+          - Transport: cli
+          - Use the ANSIBLE_PERSISTENT_COMMAND_TIMEOUT environment variable or specify a command_timeout in the ansible.cfg file.
+          -
```
Comment (wojtek0806) [inline ansible_collections/f5networks/f5_modules/plugins/doc_fragments/f5_rest_cli.py:61]: revert this change in timeout parameter, this kind of information is best suited for notes section see line 93, add this as a note

**Review by wojtek0806 — CHANGES_REQUESTED** [review]
see my comments

File: ansible_collections/f5networks/f5_modules/plugins/doc_fragments/f5_rest_cli.py (hunk 1 above)
Comment (gabetellier) [reply ansible_collections/f5networks/f5_modules/plugins/doc_fragments/f5_rest_cli.py:61]: You want me to move lines 59 and 60 ONLY to the notes section that starts on line 93, correct?

**Comment by wojtek0806** [comment]
@gabetellier  I have moved the text around , I removed the part of text referring to persistent-command-timeout, and used a link in ansible docs that explains how to set it in details

**Review by wojtek0806 — APPROVED** [review]
